omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1883 @@
+********************
+Version requirements
+********************
+
+Proposal for revised minimum version requirements
+-------------------------------------------------
+
+Minimum software versions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is a proposal to the wider community to solicit their
```

Comment:
Another possible way of integrating this section with the rest of the file is to have:

```
OME (build)	OME (deploy)   OME (next-build)	OME (next-deploy)
```
(with better names, of course) so that for every current version one could see where we are headed.